.dockerignore

```diff
@@ -0,0 +1,5 @@
+report/
```

Comment:
Since in your dockerfile you explicitly copy directories , this seems not necessary, unless you add this to be sure it's never added to the image?